backend/src/main/java/com/dbms/georgia_express/service/CustomerLoginService.java

```diff
@@ -0,0 +1,60 @@
+package com.dbms.georgia_express.service;
+
+import com.dbms.georgia_express.dto.LoginRequest;
+import com.dbms.georgia_express.dto.RegistrationRequest;
+import com.dbms.georgia_express.model.Customer;
+import com.dbms.georgia_express.model.CustomerLogin;
+import com.dbms.georgia_express.repositories.CustomerLoginRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.stereotype.Service;
+
+@Service
+public class CustomerLoginService {
+
+    @Autowired
+    private CustomerLoginRepository customerLoginRepository;
+
+    @Autowired
+    private CustomerService customerService;
+
+    private final BCryptPasswordEncoder passwordEncoder = new BCryptPasswordEncoder();
+
+    public CustomerLogin register(RegistrationRequest request) {
```

Comment:
Make sure you prevent SQL injection by securing the db. Might have to change something in the flyway.

Reply:
Aren't we save from them if we utilize prepared statements in JPA?

Reply:
I think JPA should take care for it.